analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -665,7 +675,9 @@ resource "aws_iam_policy" "s3tables_policy" {
           "s3tables:GetNamespace",
           "s3tables:GetTableBucket",
           "s3tables:GetTableBucketMaintenanceConfiguration",
-          "s3tables:GetTableBucketPolicy"
+          "s3tables:GetTableBucketPolicy",
+          "s3tables:CreateNamespace",
+          "s3tables:CreateTable"
```

Comment:
Let's add a comment above the policy saying that this is an interim solution until S3 tables full supports IRSA without relying on node instance role.